pkg/nodediscovery/nodediscovery.go

```diff
@@ -525,6 +526,13 @@ func determineFirstInterfaceIndex(instanceType string) *int {
 		if defaults.IPAMPreAllocation > max {
 			idx = 0 // Include eth0
 		}
+	} else {
+		log.WithFields(logrus.Fields{
+			"instance-type": instanceType,
+		}).Warningf(
+			"Unable to find limits for instance type, consider setting --%s=true",
+			operatorOption.UpdateEC2AdapterLimitViaAPI,
+		)
```

Comment:
I didn't realize that the flag is only for the operator...

Maybe we should clarify that in the log msg. Something like:

> "Unable to find limits for instance type, consider setting --%s=true on the Operator"

Reply:
sounds good, updated the log message and the commit message 👍